backend/internal/github/sharedclient/sharedclient.go

```diff
@@ -244,3 +244,42 @@ func (api *CommonAPI) GetRepository(ctx context.Context, owner string, repoName
 	repo, _, err := api.Client.Repositories.Get(ctx, owner, repoName)
 	return repo, err
 }
+
+func (api *CommonAPI) UpdateTeamRepoPermissions(ctx context.Context, org, teamSlug, owner, repo, permission string) error {
```

Comment:
let's specify the types for these

Reply:
This is shorthand notation for org, teamSlug, owner, repo, and permission all being strings.